### PR TITLE
 update AWS summit China blog

### PR DESCRIPTION
**Pull request by BellaXiang**
Fixes #ISSUSE_ID.

Changes proposed in this pull request:
  -

---

Before committing this PR, I'm sure that I have checked the following options:
- [ ] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [ ] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [ ] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Comment by BellaXiang** [comment]
@TeslaCN PTAL, thanks~

File: docs/blog/content/material/2023_07_07_AWS_Summit_China_2023_ShardingSphere_Highlights_and_Data_Security_Insights.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ From emerging technologies like generative AI to discussions on cloud-native tec
 
 Among the participants, the[ ShardingSphere](https://shardingsphere.apache.org/) open-source community and its PMC and[ SphereEx](https://www.sphere-ex.com/) co-founder and CTO[ Trista Pan](https://github.com/tristaZero), made their mark in the Developer Lounge, engaging with enthusiasts and sharing experiences. In this blog post, we will delve into the DevLounge's roundtable discussion on unlocking data value with open source and explore ShardingSphere's approach to data security.
 
-![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China1.jpeg") ![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China2.jpeg") ![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China3.jpeg") 
+![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China1.jpg") ![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China2.jpeg") ![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China3.jpeg")
```
Comment (TeslaCN) [inline docs/blog/content/material/2023_07_07_AWS_Summit_China_2023_ShardingSphere_Highlights_and_Data_Security_Insights.md:13]:
```suggestion
![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China1.jpg) ![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China2.jpeg) ![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China3.jpeg) 
```

File: docs/blog/content/material/2023_07_07_AWS_Summit_China_2023_ShardingSphere_Highlights_and_Data_Security_Insights.md [hunk 2]
```diff
@@ -50,9 +50,9 @@ Addressing the challenges in data value mining, Trista highlighted data security
 
 At the event, ShardingSphere's lightweight open-source distributed SQL transaction & query engine garnered significant attention. Database developers, operations engineers, and architects showed great interest in ShardingSphere's technical framework, implementation, feature highlights, and its joint solution with AWS Aurora. Many expressed a desire to join the ShardingSphere community to gain hands-on experience and expand their knowledge.
 
-![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China7.jpeg") ![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China8.jpeg")
+![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China7.jpg") ![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China8.jpg")
```
Comment (TeslaCN) [inline docs/blog/content/material/2023_07_07_AWS_Summit_China_2023_ShardingSphere_Highlights_and_Data_Security_Insights.md:53]:
```suggestion
![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China7.jpg) ![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China8.jpg)
```

File: docs/blog/content/material/2023_07_07_AWS_Summit_China_2023_ShardingSphere_Highlights_and_Data_Security_Insights.md [hunk 3]
```diff
@@ -50,9 +50,9 @@ Addressing the challenges in data value mining, Trista highlighted data security
...
 
-![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China9.jpeg") ![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China10.jpeg")
+![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China9.jpg") ![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China10.jpg")
```
Comment (TeslaCN) [inline docs/blog/content/material/2023_07_07_AWS_Summit_China_2023_ShardingSphere_Highlights_and_Data_Security_Insights.md:55]:
```suggestion
![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China9.jpg) ![img](https://shardingsphere.apache.org/blog/img/AWS_Summit_China10.jpg)
```

**Review by TeslaCN — APPROVED** [review]

**Pull request merged**